src/applications/vaos/new-appointment/components/DateTimeSelectPage/WaitTimeAlert.jsx

```diff
@@ -33,7 +82,7 @@ export const WaitTimeAlert = ({
   if (momentPreferredDate.isValid() && momentNextAvailableDate.isValid()) {
     const showUrgentCareMessage = today.isSame(momentPreferredDate, 'day');
 
-    if (showUrgentCareMessage) {
+    if (showUrgentCareMessage && nextAvailableApptDate) {
```

Comment:
Add a boolean to replace the variable string `nextAvailableApptDate`
```suggestion
  const hasNextAvailableApptDate = !!nextAvailableApptDate;
  if (showUrgentCareMessage && hasNextAvailableApptDate) {

```

Reply:
Done.